.github/labeler.yml

```
@@ -29,8 +29,6 @@ appleOS:
               .github/workflows/macos.yml,\
               lib/config-mac.h,\
               lib/macos*,\
```

```suggestion
              lib/macos*\
```
Drop closing comma just in case. The other filter expressions do this.